### DISCUSSION TITLE
How to remove label from multiselect

### DISCUSSION THREAD

**Discussion by lytechxdigital** [react-querybuilder/react-querybuilder#761] [Q&A]
i try to remove but showing 
![image](https://github.com/user-attachments/assets/a86b034d-eade-47ba-ba77-85c825c56639)

![image](https://github.com/user-attachments/assets/b9e66fea-43d4-4723-998c-fae1edf0d8b3)

my code is 
`const industryFiltterData = industries.map((group, idx) => ({
    // label: group.label, // Unique identifier for each group
    options: group?.children?.map((category) => ({
      name: category.value,
      label: category.label,
    })),
  }));`

thanks

**Comment by jakeboone02** [comment]
Those labels show up when you use an `OptionGroup[]` instead of `Option[]`.

Short answer: see https://react-querybuilder.js.org/docs/tips/option-lists.

Longer answer:

`OptionGroup[]` has a structure like this:

```
- OptionGroup
  - Option
  - Option
- OptionGroup
  - Option
  - Option
```

...whereas `Option[]` has a flat structure:

```
- Option
- Option
- Option
- Option
```

The "instruments" list from the demo is defined [here](https://github.com/react-querybuilder/react-querybuilder/blob/0cf4c4e7c81aba97038c597f940d8e13c12a8372/website/src/pages/demo/_constants/musicalInstruments.ts). It ends up looking something like this:

```ts
[
  {
    label: "Percussion instruments", // <-- this shows up as a group label
    options: [
      { name: "Clapstick", label: "Clapstick" },
      { name: "Cowbell", label: "Cowbell" },
      // ...
    ]
  },
  {
    label: "Membranophones", // <-- this shows up as a group label
    options: [
      { name: "Barrel drum", label: "Barrel drum" },
      { name: "Bass drum", label: "Bass drum" },
      // ...
    ]
  }
]
```

To make it a flat structure, move all the nested option up to the top level and remove the "options" groups:

```ts
[
  { name: "Clapstick", label: "Clapstick" },
  { name: "Cowbell", label: "Cowbell" },
  // ...
  { name: "Barrel drum", label: "Barrel drum" },
  { name: "Bass drum", label: "Bass drum" },
  // ...
]
```
